app/controllers/gems_adoptions_controller.rb

```diff
@@ -0,0 +1,21 @@
+class GemsAdoptionsController < ApplicationController
+  
+  def index
+    @gems_adoptions = GemsAdoption.all
+  end
+
+  def create
+    @gems_adoption = GemsAdoption.new
+    @gems_adoption.gem = params[:name]
+    @gems_adoption.description = params[:info]
+    @gems_adoption.user_id = current_user.id
+    @gems_adoption.save
+    redirect_to gem_path(id: params[:name])
+  end
+
+  def destroy
+    GemsAdoption.find(params[:id]).destroy
+    redirect_to gem_path(id: params[:name])
```

Comment:
Aren't we redirecting the user to the parth of the `GemsAdoption` we just destroyed here?